README: New topics on forum (#286)
ansible-community/community-topics: README.md

[![Discuss on Matrix at #community:ansible.com](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20on%20Matrix%20at%20%23community:ansible.com&logo=matrix)](https://matrix.to/#/#community:ansible.com)

This repository historically contained topics to be discussed and voted by the Ansible Community (all interested people) and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.

> The **scope** of the topics was changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.

Reviewer: ```suggestion
The **scope** of the topics were changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.
```

Reviewer 2: this is a bit tricky one:)

Reviewer: Maybe let's keep is/are, since that's still the scope of the topics - both here and on the forum.

Author: I could change it again, but what be correct? Would it be `is` because it's only one scope? Or `are` because of `changes` (plural!)?

I tend to `is` but I'm not a native speaker... maybe someone who is could advise.

Reviewer 3: The scope of the topics `is` changes...

`In` the scope of the topics `are` changes... (rephrased, Changes are in the scope of the topics)

I would say, The scope of the topics `includes` (or `encompasses`) changes...

Author: Thanks for the suggestion @cybette!

@Andersson007 @felixfontein Does my [last commit](https://github.com/ansible-community/community-topics/pull/286/commits/df32fc39d86109dbdb0e41cd8a5e0b32042214ee) look OK to you, should we resolve this conversation?

Reviewer: It looks good to me!

We decided to discuss those topics in the [Ansible forum](https://forum.ansible.com) in the future. However, we will not move the existing discussions to the forum and will keep to work on them here.

From now on, if you want to discuss something that satisfies the scope then [create a new topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=community-wg) tagged with `community-wg` under the `Project Discussions` category or [comment on an existing one](https://forum.ansible.com/search?expanded=true&q=%23project%20tags%3Acommunity-wg).

> 

Reviewer: Maybe add a link to the votes as well?
```suggestion

All active votes in the Ansible Forum can be found using [the intersection of the `community-wg` and `active-vote` tags](https://forum.ansible.com/tags/intersection/community-wg/active-vote).

```

[![Discuss on Matrix at #community:ansible.com](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20on%20Matrix%20at%20%23community:ansible.com&logo=matrix)](https://matrix.to/#/#community:ansible.com)

> This repository historically contained topics to be discussed and voted by the Ansible Community (all interested people) and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.

Reviewer 4: ```suggestion
This repository contains historical topics for the Ansible Community (all interested people) and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) to discuss and vote on asynchronously.
```

The **scope** of the topics includes changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.

We decided to discuss those topics in the [Ansible forum](https://forum.ansible.com) in the future. However, we will not move the existing discussions to the forum and will keep to work on them here.

> From now on, if you want to discuss something that satisfies the scope then [create a new topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=community-wg) tagged with `community-wg` under the `Project Discussions` category or [comment on an existing one](https://forum.ansible.com/search?expanded=true&q=%23project%20tags%3Acommunity-wg).

Reviewer 4: ```suggestion
From now on, if you want to discuss something that satisfies the scope then [create a new topic](https://forum.ansible.com/new-topic?category=project&tags=community-wg) tagged with `community-wg` under the `Project Discussions` category or [comment on an existing one](https://forum.ansible.com/tags/c/project/7/community-wg).
```

All active votes in the Ansible Forum can be found using [the intersection of the `community-wg` and `active-vote` tags](https://forum.ansible.com/tags/intersection/community-wg/active-vote).

> You do **NOT** need any special status (for example, to be a Steering Committee member) to create the topics or take part in the discussions asynchronously in the topics or in the meetings (see below). Any opinions are equally welcome and appreciated!

Reviewer 5: Since we're updating this page, I'd change this wording to be more positive:
```suggestion
Community members may create topics, take part in asynchronous discussions, and vote! All opinions are equally welcome and appreciated. You do **NOT** need to be a Steering Committee member or to have any other special status to create topics or take part in the discussions asynchronously in the topics or in the meetings (see below). 
```
